libs/sdk-ui-tests-e2e/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# (C) 2021 GoodData Corporation
+
+FROM harbor.intgdc.com/tools/gdc-frontend-node:node-12.16.1-yarn-1.22.4 as build-stage
+
+ARG NPM_TOKEN
```

Comment:
The stuff around NPM_TOKEN should not be needed in SDK, we do not use any private packages (as opposed to the app you took insiration from).

Reply:
This Dockerfile gonna be changed in the next phase, until now, it is just a copy of the original KD Dockerfile, which is using the token. We will have it different for a CI integration completely.

Reply:
Ok, fair enough